test/address_tests.c

```diff
@@ -16,22 +16,25 @@ void test_address()
 {
     size_t privkeywiflen = 100;
     char privkeywif[privkeywiflen];
-    char privkeyhex[100];
-    u_assert_int_eq(generatePrivPubKeypair(privkeywif, privkeyhex, false), true)
+    char p2pkh_wif[100];
```

Comment:
Do we want to make the p2pkh_wif field 100 characters wide?  Should be a max of 36.